Add watergate documentation (#35535)
home-assistant/home-assistant.github.io: source/_integrations/watergate.markdown

- Watergate Sonic Wi-Fi

## Unsupported devices

The following devices are not supported by the integration:

- Watergate Sonic Pro

## Supported functionality

### Entities

The Watergate integration provides the following entities.

#### Sensors

- **Water meter volume**
  - **Description**: Water volume used by this device for the entire lifetime of a Sonic device.
  - **Remarks**: It can be used in the Energy Dashboard.

- **Water flow rate**
  - **Description**: The current flow rate of water through the device.
  - **Remarks**: Useful for monitoring real-time water usage.

- **Water pressure**
  - **Description**: The current water pressure in the system.
  - **Remarks**: Can be used to detect potential issues in the water supply.

- **Water temperature**
  - **Description**: The current temperature of the water.
  - **Remarks**: Useful for monitoring and ensuring safe water temperatures.

#### Valves

- **Water valve state**
  - **Description**: The current state of the water valve (open/closed).
  - **Remarks**: It is automatically updated when the valve state is changed.

## Data updates

The Watergate integration fetches data from the Sonic device every 2 minutes.

> Thanks to the webhook option, Sonic will provide live telemetry every second when water is flowing directly to Home Assistant.

Author: It is not required I guess.

---
title: Watergate
description: Instructions on how to integrate Watergate within Home Assistant.
ha_category:
  - Sensor
  - Valve
  - Binary Sensor
  - Number

>   - Water management

Reviewer: You only have Valve here as well

ha_release: '2025.1'
ha_iot_class: Local Push
ha_config_flow: true
ha_codeowners:
  - '@adam-the-hero'
ha_domain: watergate
ha_platforms:
  - valve
---

The **Watergate** {% term integration %} integrates your Watergate Devices (currently Sonic) to your Home Assistant.
With this integration you are able to:

- control your valve
- configure auto shut off parameters
- gather auto shut off parameters

> - check diagnostics of the device

Reviewer: Please make sure the PR matches the current features implemented. As in, we should be able to cut a release at this moment and the documentation should be correct and not describing future features

---
title: Watergate
description: Instructions on how to integrate Watergate within Home Assistant.
ha_category:
  - Sensor
  - Valve
  - Binary Sensor
  - Number
  - Water management

> ha_release: '2024.11'

Reviewer: ```suggestion
ha_release: '2025.1'
```

ha_iot_class: Local Push
ha_config_flow: true
ha_codeowners:

>   - '@watergate-ai'

Reviewer: ```suggestion
  - '@adam-the-hero'
```

ha_domain: watergate
ha_platforms:

>   - diagnostics
>   - sensor
>   - valve
>   - binary_sensor
>   - number

Reviewer: ```suggestion
  - valve
```

> - control your valve
> - monitor telemetry
> - gather auto shut off reports
> - check diagnostics of the device

Reviewer 2: ```suggestion
- Control your valve
- Monitor telemetry
- Gather auto shut-off reports
- Check diagnostics of the device
```

## Prerequisites

> - You need to have Sonic device
> - Local API feature must be enabled in application
> - Network Traffic between Home Assistant Server and Sonic must be unlocked (Server uses HTTP protocol on 80 port).

Reviewer 2: ```suggestion
- You need to have a Sonic device.
- Local API feature must be enabled in application.
- Network traffic between Home Assistant and Sonic must be possible. (server uses HTTP protocol on 80 port).
```

PS: I'm really doubting the latter point is an pre-requisite we should mention here.

{% important %}

> Go to Watergate application and enable Local API in specific device's settings.

Reviewer 2: This is a duplicate of the second point?

Author: Yes it is. I've removed it I think there is no point to write 101 instructions for that.
